src/components/MobileMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled';
+import Link from 'next/link';
+import React, { FC } from 'react';
+import { useLockBodyScroll, useToggle } from 'react-use';
+import { CTA } from '.';
+import { MenuIcon } from './icons';
+
+const textAnimation = `
+  opacity: 0;
+
+  .visible & {
+    opacity: 1;
+    transition: opacity 500ms ease 100ms;
+  }
+`;
+
+const Menu = styled.div`
+  background-color: ${({ theme }) => theme.colors.workThumbnailBg};
+  height: 100%;
+  left: 0;
+  position: fixed;
```

Comment:
```suggestion
  opacity: 0;
```
begin opacity at zero before updated transition below.